installer_scripts/install-yb-voyager

```diff
@@ -27,7 +27,7 @@ LOG_FILE=/tmp/install-yb-voyager.log
 
 # commit hash in main branch after release 23.2
 # issues #489 #492 #247 in ora2pg were fixed after this change
-ORA2PG_VERSION="3e27abba5939d5796419a1d835d6d4d0b2d5e01d"
+ORA2PG_VERSION="f08d47a4e5c20ff7d348e535ed05ab571b4bb350"
 
```

Comment:
We might have to update it to this `7b8f018352fb24ce46e368358178e6b542381cfd` as  there are these three fixes which also need to picked up and this commit other cover all these three fixes.
1. https://github.com/yugabyte/yb-voyager/issues/574
2. https://github.com/yugabyte/yb-voyager/issues/511
3. https://github.com/yugabyte/yb-voyager/issues/573

CC: @shubham-yb

Reply:
The commit you have mentioned is after the commit past which ora2pg breaks in our automation tests (`8b64f1a543c9e63d3eff80d43b83a74298bfb104`). I have taken the commit right before this point to maximise the issues fixed within ora2pg

Reply:
Oh okay, I didn't knew the commit for our fix is breaking some other things. Cool then we are good here.